docs/spec/auth/subkeys_proposal.md

```diff
@@ -0,0 +1,156 @@
+# Subkeys ADR
+
+## Abstract
+
+Currently, a `StdAccount` has only one public key assoiciated with it, and all transactions that are made with that key must be signed by that one public key. This is problematic because the security level required by different types of transactions executable by an account may be different. For example, an account may want to keep the key that has the ability to do `SendMsg`s in cold storage, but wants to keep a key that can vote on governance proposals on their phone.
```

Comment:
```suggestion
Currently, a `StdAccount` has only one public key assoiciated with it, and all transactions that are made with that key must be signed by that one public key. This is problematic because the security level required by different types of transactions executable by an account may be different. For example, a user may want to keep the key that has the ability to send their funds in cold storage, but keep a key that can vote on governance proposals on their phone.
```